### ISSUE TITLE
Extra plugin icon in toolbar

### ISSUE THREAD

**Issue by Tich1212** [HydrologicEngineeringCenter/SimpleRTSPlugin#8]
It appears that upon registering the plugin, the CAVI adds a simple action button to the programs toolbar with the short name of the plugin, "Simple". Then when I go to add the button how I prefer (anchor), I end up with two buttons for the same plugin. How do I stop the Simple button from being added?

![image](https://user-images.githubusercontent.com/57423836/75296741-ec7c0e80-57e2-11ea-9513-f14afb4f3c5f.png)

**Comment by HenryGeorgist** [comment]
You are adding the image on line 63 to the toolbar.  You are adding it to the menu item on line 67. Comment out one or the other and you will only have one or the other

**Comment by Tich1212** [comment]
Let me try again,  I don't want two toolbar buttons which is what I have right now (Forget the Tools menu item).

![image](https://user-images.githubusercontent.com/57423836/75299953-513b6700-57eb-11ea-97f7-2d2eb96eaa2a.png)

**Comment by HenryGeorgist** [comment]
Oh wait, you mean why does simple and the anchor appear on the toolbar.  I would have to look into browser action or ask Mark Ackerman

**Comment by Tich1212** [comment]
Right. I believe it is occurring when I register the plugin on line 54. I can comment everything else out and still get the "Simple" button. 

If I can find the source code for the register method I can determine this for sure.

**Comment by HenryGeorgist** [comment]
Comment out line 55. Addtotoolstoolbar.
